services/admin/src/server.ts

```diff
@@ -379,12 +380,58 @@ export function buildApp({ store }: { store: Store }): Application {
       return;
     }
 
-    const { contestId } = parseQueryResult.ok();
+    const { contestId, status } = parseQueryResult.ok();
     response.json(
-      store.getWriteInAdjudicationSummary({ electionId, contestId })
+      store.getWriteInAdjudicationSummary({
+        electionId,
+        contestId,
+        status,
+      })
     );
   });
 
+  app.get<
+    Admin.GetWriteInAdjudicationTableUrlParams,
+    Admin.GetWriteInAdjudicationTableResponse,
+    Admin.GetWriteInAdjudicationTableRequest
+  >(
+    '/admin/elections/:electionId/contests/:contestId/write-in-adjudication-table',
```

Comment:
I thought we were trying to avoid nesting params?

Reply:
Where it's possible to infer the election ID I haven't been including it in the URL, yeah. Here both the election ID and contest ID must be specified, so I put them both in the URL.